scripts/docker-run-tests.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
I guess it could, but I also think there's a point to be able to break things up into distinct steps (`pre_install`, `install`, `script`, etc.) as done in the Travis file.